lib/kernel.coffee

```diff
@@ -15,7 +15,7 @@ class Kernel
         console.log 'Kernel spec:', @kernelSpec
         console.log 'Kernel configuration:', @config
         console.log 'Kernel configuration file path:', @configPath
-        @language = @kernelSpec.grammarLanguage
+        @language = @kernelSpec.language
```

Comment:
After some more thought and since `Kernel` gets a `grammar`, I think this line should be `@language =  KernelManager::getLanguageFor grammar`.

This would also suggest that  `getLanguageFor` should be a static method (I would leave that for another PR because `getLanguageFor` is used everywhere).